debian/copyright

```diff
@@ -0,0 +1,23 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: neofs-node
+Upstream-Contact: tech@nspcc.ru
+Source: https://github.com/nspcc-dev/neofs-node
+
+Files: *
+Copyright: 2018-2022 NeoSPCC (@nspcc-dev), contributors of neofs-node project 
+           (https://github.com/nspcc-dev/neofs-node/blob/master/CREDITS.md)
+
+License:
+
+   This program is free software: you can redistribute it and/or modify it
```

Comment:
And IIRC it should be idented with one space exactly.

Reply:
I can't find indentation rules :-( but ok for me

Reply:
Section 4 of https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/ references 5.1 of https://www.debian.org/doc/debian-policy/ch-controlfields#s-controlsyntax.